scripts/intfutil

```diff
@@ -863,10 +866,86 @@ class IntfFecStatus(object):
                 table.append((key, oper_fec, admin_fec))
         return table
 
+
+# ========================== interface path-tracing logic ==========================
+
+header_path_tracing = ['Interface', 'Alias', 'Oper', 'Admin', 'PT Interface ID', 'PT Timestamp Template']
+
+class IntfPathTracing(object):
+
+    def __init__(self, intf_name, namespace_option, display_option):
+        """
+        Class constructor method
+        :param self:
+        :param intf_name: string of interface
+        :return:
+        """
+        self.db = None
+        self.config_db = None
+        self.intf_name = intf_name
+        self.table = []
+        self.multi_asic = multi_asic_util.MultiAsic(
+            display_option, namespace_option)
+
+        if intf_name is not None and intf_name == SUB_PORT:
+            self.intf_name = None
+
+    def display_intf_path_tracing(self):
+        self.get_intf_path_tracing()
+
+        # Sorting and tabulating the result table.
+        sorted_table = natsorted(self.table)
+        print(tabulate(sorted_table, header_path_tracing, tablefmt="simple", stralign='right'))
+
+    def generate_intf_path_tracing(self):
+        """
+            Generate interface path-tracing output
+        """
+
+        i = {}
+        table = []
+        key = []
+
+        intf_fs = parse_interface_in_filter(self.intf_name)
+        #
+        # Iterate through all the keys and append port's associated state to
+        # the result table.
+        #
+        for i in self.appl_db_keys:
+            key = re.split(':', i, maxsplit=1)[-1].strip()
+            if key in self.front_panel_ports_list:
+                if self.multi_asic.skip_display(constants.PORT_OBJ, key):
+                    continue
+
+                if self.intf_name is None or key in intf_fs:
+                    table.append((key,
+                        appl_db_port_status_get(self.db, key, PORT_ALIAS),
+                        appl_db_port_status_get(self.db, key, PORT_OPER_STATUS),
+                        appl_db_port_status_get(self.db, key, PORT_ADMIN_STATUS),
+                        appl_db_port_status_get(self.db, key, PORT_PATH_TRACING_INTERFACE_ID),
+                        appl_db_port_status_get(self.db, key, PORT_PATH_TRACING_TIMESTAMP_TEMPLATE)))
+        return table
+
+    @multi_asic_util.run_on_multi_asic
+    def get_intf_path_tracing(self):
+        self.front_panel_ports_list = get_frontpanel_port_list(self.config_db)
+        self.appl_db_keys = appl_db_keys_get(self.db, self.front_panel_ports_list, None)
+        self.get_raw_po_int_configdb_info = get_raw_portchannel_info(self.config_db)
+        self.portchannel_list = get_portchannel_list(self.get_raw_po_int_configdb_info)
```

Comment:
why port-channel?

Reply:
The port channel list was defined but never used, now I removed it.

Reply:
@kperumalbfn The port channel list was defined but never used, now I removed it.